libnavigation-core/src/main/java/com/mapbox/navigation/core/routealternatives/RouteAlternativesController.kt

```diff
@@ -115,24 +169,31 @@ internal class RouteAlternativesController constructor(
         }
 
         override fun onError(message: String) {
-            logE(TAG, Message("Error: $message"))
+            observers.forEach {
+                // NN should expose origin of a failed alternatives request and the used URL,
+                // refs https://github.com/mapbox/mapbox-navigation-native/issues/5401
+                // and https://github.com/mapbox/mapbox-navigation-native/issues/5402
```

Comment:
Capturing from @DenisPryt 
> RouteAlternative.route.routerOrigin: optionalrouter::RouterOrigin
Empty optional means that route from setRoutes method (unknown origin)

Reply:
This is not yet available in Nav Native, it was just merged and I assume we'll be able to address this with the upcoming release.